js/build.js

```diff
@@ -0,0 +1,77 @@
+Fliplet.Widget.instance({
+  name: 'image-component',
+  displayName: 'Image component',
+  render: {
+    template: [
+      '<span class="image-component-container"></span>'
```

Comment:
Review why this is needed. `$(image.$el)` probably gives you the right reference point.